lib/docker.js

```diff
@@ -32,24 +34,43 @@ function buildImage(dockerFile) {
   return imageName;
 }
 
+/**
+ * Find a file that exists on all projects so we can test if Docker can see it too
+ * @param {string} servicePath
+ * @return {string} file name
+ */
+function findTestFile(servicePath) {
+  if (fse.pathExistsSync(path.join(servicePath, 'serverless.yml'))) {
+    return 'serverless.yml';
+  }
+  if (fse.pathExistsSync(path.join(servicePath, 'serverless.yaml'))) {
+    return 'serverless.yaml';
```

Comment:
Did serverless drop support for `serverless.json`?

Reply:
I was not aware that's an option. Let me add that.